docs/index.md

```diff
@@ -557,28 +557,28 @@ You can also call these functions with arguments as described in Faker's [docume
 country: ${{fake.country_code(representation='alpha-2')}}
 ```
 
-### International Fakes (syntax still under development)
+### International Fakes
 
-You can specify internationally appropriate fakes for many different kind of names (e.g. person, company) like this:
+You can specify internationally appropriate fakes for many different kind of names (e.g. person, company) by setting the snowfakery_locale this:
 
 ```yaml
-- object: Viking
+- var: snowfakery_locale
+  value: no_NO
+- object: person
   fields:
-    Name:
-      i18n_fake:
```

Comment:
@prescod Is this syntax no longer supported? If so that should probably be called out in the release notes.